documentation/report.md

```diff
@@ -0,0 +1,238 @@
+# <div align="center">Pace Group C - Requirements and Design Report</div>
+
+
+## <div align="center">High-Level Project Description with Correct Identification of User Groups</div>
+#### General Description
+The software we are building is to create a web-based Mentorship platform that would allow users to register as a mentor or a mentee. In addition, users from the organization using our product can register and sign in as administrators. After registering, a mentor or mentee are required to fill a mentor-ship form for identification. Then, the platform shows viewable calendar for availability of mentors from which mentees can request for a meeting, and, on their version of the site, mentors can view and manage mentees' mentorship requests. 
+
+#### Function requirement description
+* Mentor:
+System requires mentors to create their own accounts and fill out the sign-up form in order to approve mentee's matching requests via emails. In addition, System requires mentors to provide email address and availability of their schedule for mentees to book an appointment.
+
+* Mentee:
+System does not require mentees to have their own account. Instead, they are required to fill out the sign-up form in order to view availability of mentors they are going to match with. Also, they need to provide contact info while filling the form.
+
+* Admins:
```

Comment:
@wrc9817  Since we have information about Admins from the client notes, can you add in something along the lines of: "The role of an Admin is to decide how to host the form (either publicly or internally through email invite), edit the embedded iFrame form, and oversee Mentors."

Reply:
Doing and Thank you for reminding me1

Reply:
Do you think we need any methods to remind us that the admin part should be updated and organization button should be deleted in next step? Like a todo for next step?